ci-operator/step-registry/addon/rosa/addon-rosa-workflow.yaml

```diff
@@ -8,7 +8,6 @@ workflow:
       ENABLE_AUTOSCALING: "false"
       REPLICAS: "2"
       MULTI_AZ: "false"
-      PARALLEL: "true"
     post:
```

Comment:
please re-add `PARALLEL: "true"` - this is the default we'd like in most cases; set to "" in config file.